.github/workflows/pull-request.yml

```
@@ -421,22 +421,29 @@
-      - uses: actions/setup-node@v4
+      - name: Checkout the Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
         with:
-          node-version: '22.x'
-          cache: yarn
-          cache-dependency-path: 'docs/'
-      # Run Docusaurus in the ./docs directory
+          node-version: '20.x' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)